proto/dydxprotocol/blocktime/query.proto

```diff
@@ -22,7 +22,9 @@ service Query {
 
   // Queries all recorded downtime info.
   rpc AllDowntimeInfo(QueryAllDowntimeInfoRequest)
-      returns (QueryAllDowntimeInfoResponse);
+      returns (QueryAllDowntimeInfoResponse) {
+    option (google.api.http).get = "/dydxprotocol/v4/blocktime/all_downtime_info";
```

Comment:
Added an HTTP query since imo this information is helpful to expose in an HTTP endpoint. LMK if anyone would prefer i remove from this pr